packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -36,7 +37,15 @@ const CMSPage = props => {
             return <RichContent html={data.cmsPage.content} />;
         }
 
-        return <CategoryList title="Shop by category" id={2} />;
+        return (
+            <>
+                <Meta
+                    name="description"
+                    content={data.cmsPage.meta_description}
```

Comment:
@davemacaulay do you guys want this in the above conditional?

Reply:
It makes sense to me to have it for all resolved pages, the above conditional was to ensure the category list shows up on instances without Page Builder.

Reply:
Right -- I'm considering just adding this to all pages but this specific changeset was supposed to address the lighthouse score dinging us for not having the description on the root page (which happens to be CMS).

Reply:
It should then be included on all CMS pages, correcting meta data for all pages is more beneficial then a lighthouse score fix imo.

Reply:
Agreed :) I went ahead and added it to everything.